src/test/java/com/artipie/SettingsFromPathTest.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * The MIT License (MIT) Copyright (c) 2020-2021 artipie.com
+ * https://github.com/artipie/artipie/LICENSE.txt
+ */
+package com.artipie;
+
+import com.amihaiemil.eoyaml.Yaml;
+import com.google.common.io.Files;
+import java.io.IOException;
+import java.nio.file.Path;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.core.IsInstanceOf;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+
+/**
+ * Test for {@link SettingsFromPath}.
+ * @since 0.3
+ */
+class SettingsFromPathTest {
+
+    @Test
+    void createsSettings(final @TempDir Path temp) throws IOException {
```

Comment:
@olenagerasimova probably to add test when it takes default settings file

Reply:
@genryxy I cannot, as file storage is set to `/var/artipie/repo` in `example/artipie.yaml`, this path does not exist generally